CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ file(STRINGS version.txt EVT_VERSION)
 
 project(EVT
     VERSION ${EVT_VERSION}
-    LANGUAGES CXX C
+    LANGUAGES CXX C ASM
```

Comment:
Why do we need to compile assembly like this, while we don't do that for the other assembly files we have?